apps/fabric-website/src/pages/Styles/FabricIconsPage/docs/web/FabricIconsUsage.md

```diff
@@ -41,9 +43,13 @@ const MyIconButton = () => <IconButton iconProps={{ iconName: 'Add' }} title="Ad
 ReactDOM.render(<MyIconButton />, document.body.firstChild);
 ```
 
-### Fluent UI React: SVG-based icons
+### Fluent UI React: SVG-based icons (open source as of version 8)
```

Comment:
Since the whole text of the header gets used when generating the anchor link, it's probably best to either leave this text as-is (the suggestion) or update that anchor's URL referenced in FabricIconsPage.tsx.
```suggestion
### Fluent UI React: SVG-based icons
```